### DISCUSSION TITLE
[Question][Hw5] git wrong file

### DISCUSSION THREAD

**Discussion by ksy980418** [prosyslab-classroom/cs348-information-security#631] [Q&A]
When I tried to test the static analyzer, I found that there is no example1.expected ... etc
Instead, I have the only example1.sign.expected, example2.sign.expected, example3.sign.expected.
How can I get the correct files?

**Comment by RiceBiscuits** [comment]
Please refer to this document(#131) and modify the script.

**Comment by ksy980418** [comment]
I already did that one.

I have only 3 expected files and the names are different.

**Comment by RiceBiscuits** [comment]
You should refer to this document (https://github.com/prosyslab-classroom/cs448-2021-spring/issues/130#issuecomment-846989010) too.

**Comment by ksy980418** [comment]
Where do I have to add that modified one?
And Can I just change the name of the expected files for 1,2,3?

`new file mode 100644
index 0000000..ac488ee
--- /dev/null
+++ b/test/example4.expected
@@ -0,0 +1 @@
+Potential Tainted-flow @ example4.c:main:5:3 (call void @sink(i32 %call), !dbg !14)
diff --git a/test/example5.expected b/test/example5.expected
new file mode 100644
index 0000000..ccb4aef
--- /dev/null
+++ b/test/example5.expected
@@ -0,0 +1 @@
+Potential Tainted-flow @ example5.c:main:12:3 (call void @sink(i32 %y.0), !dbg !23)
`

**Comment by RiceBiscuits** [comment]
1. Modify your script
2. You can create two files (example4.expected, example5.expected) under the test directory.
